reference/docs-conceptual/PowerShell-Core-Support.md

```diff
@@ -33,25 +33,48 @@ Eventually, we expect PowerShell Core will adopt the "long-term servicing" appro
 
 ## Supported platforms
 
-PowerShell Core is officially supported on the following platforms:
-
-* Windows 7, 8.1, and 10
-* Windows Server 2008 R2, 2012 R2, 2016
-* [Windows Server Semi-Annual Channel][semi-annual]
-* Ubuntu 14.04, 16.04, and 17.04
-* Debian 8.7+, and 9
-* CentOS 7
-* Red Hat Enterprise Linux 7
-* OpenSUSE 42.2
-* Fedora 27, 28
-* macOS 10.12+
-
-Our community has also contributed packages for the following platforms,
-but they are not officially suppported:
-
-* Arch Linux
-* Kali Linux
-* AppImage (works on multiple Linux platforms)
+Please see the following table to see what platform the version of PowerShell Core you are using is officially supported.
+
+Our community has also contributed packages for some platforms,
+but they are not officially supported.
+These packages are marked as `Community` in the table.
+
+Platforms listed as `Experimental` are not officially supported, but is available for experimentation of feedback.
```

Comment:
"...but are available for experimentation and feedback."

Reply:
Thanks